core/templates/dev/head/pages/exploration_player/InputResponsePairDirective.js

```diff
@@ -23,7 +23,8 @@ oppia.directive('inputResponsePair', [
       scope: {
         data: '=',
         oppiaAvatarImageUrl: '&',
-        profilePicture: '&'
+        profilePicture: '&',
+        inputResponsePairId: '=',
```

Comment:
Try to use `getInputResponsePairId: '&inputResponsePairId'` if you can. One-way access is better.